index.js

```diff
@@ -182,7 +182,7 @@ const ruleFunction = (expectation, options, context) => {
           insertText +
           rawData.substring(difference.offset + deleteText.length)
         );
-      }, root.source.input.css);
+      }, source);
```

Comment:
it looks like the prettier always runs on the raw input data, and not on the current version of the code (some rules may have done their job before the prettier).